### PR TITLE
add: button and image page

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,
I have added a button and image page to the documentation site.
Please review and suggest.

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
Looks good

File: pages/button.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+        <section class="component-section">
+            <h4>Floating Buttons</h4>
+            <p>Floating button is fixed at the bottom-right corner. You customize it by adding different color and icon.</p>
+            <div class="component-example-container">
+                <i class="fas fa-plus button danger floating"></i>
+            </div>
+            <pre class="component-code-snippet">
+                <code class="language-html">
+                    <img class="img-responsive img-squared" src="/images/cup1.jpg" alt="responsive-image"/>
```
Comment (mikesch803) [inline pages/button.html]: there should be floating icon code i guess

Comment (ApurvaSawant11) [reply pages/button.html]: Ohh! Thanks. I have changed it

Comment (mikesch803) [reply pages/button.html]: looks good now

**Review by mikesch803 — APPROVED** [review]

**Pull request merged**